deploy/docs/Best_Practices.md

```diff
@@ -511,6 +530,64 @@ You can find more information on the `grep` filter plugin in the
 Refer to our [documentation](v1_conf_examples.md) for other examples of how you can
 customize the fluentd pipeline.
 
+## Modifying logs in Fluentd
+
+You can redact log messages in order to e.g. prevent sending sensitive data like passwords.
+
+In order to do that [record_transformer filter][record_transformer] can be used.
+Please consider the following configuration:
+
+```yaml
+fluentd:
+  logs:
+    containers:
+      extraFilterPluginConf: |-
+          <filter containers.var.log.pods.*_*_*.test-container.*.log containers.var.log.containers.*_*_test-container-*.log>
```

Comment:
In this section it is not mentioned that this filter is only for container with name `test-container`.

Reply:
added